### ISSUE TITLE
 Invite member - Search is unable to validate when more than one email used belong to member

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#29494]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.3.83-5

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to Settings > Workspaces > any workspace > Member > Invite
3. Invite two members (A and B)
4. Return to Invite page
5. Enter the email of User A
6. Enter the email of User A and B

## Expected Result:
The search field should be able to validate when both the emails used belong to the workspace member, as it does when one member email is searched

## Actual Result:
The search result shows invalid email rather than saying User A and B are member of the workspace

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

 

https://github.com/Expensify/App/assets/78819774/033e85f2-8a6e-476a-8d18-501100756176

</details>

<details>
<summary>MacOS: Desktop</summary>

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by jasperhuangg** [comment]
Ahh it's because we didn't handle a case, I can take this

**Comment by jasperhuangg** [comment]
Don't think we need to block deploy on this tough

**Comment by jasperhuangg** [comment]
This seems to be a regression from https://github.com/Expensify/App/pull/29445

@Samueljh1 please look into this a raise a follow-up PR

**Comment by Samueljh1** [comment]
> This seems to be a regression from #29044. @Samueljh1 Please raise a followup PR to fix this. The validation should be flexible in handling whether invalid/excluded emails are entered in the search input [here](https://github.com/Expensify/App/blob/cb4b8a00705047085c3bcc287f80459ecbb44a2f/src/pages/workspace/WorkspaceInvitePage.js#L229-L238)
> 
> For instance, if you enter an invalid email and a valid email: `invalid@email.com, valid@email.com`
> 
> `valid@email.com` should still appear as an option to invite, but we should show an error message saying `invalid@email.com` isn't valid.

I considered doing this while I submitted the PR, but I decided not to due to the significant level of complexity added. This feature was requested so that emails could be added in bulk. In the case of invalid/already invited emails, the code simply ignores them and only displays valid ones in the search result. This issue is the result of an edge case where everyone in the search term is a member already (or has an invalid email).
- Suppose we have the search term: ‘admin@expensify.com, alreadyinvited@gmail.com’. Which error message do we display? One is an invalid email, and the other is an email already invited. We would probably have to create a new message along the lines of ‘None of the emails specified can be invited’
- Secondly, in the context of a corporate situation, possibly hundreds of emails could be pasted in at once. Do we want to clog up the window with a huge error message listing all the already invited emails instead of simply ignoring them from the search results?

These were the reasons I left the error messaging handling to the way they are now. I think the most suitable approach should be discussed first before I make any further changes to the PR.
